src/store/transaction/mutations.ts

```diff
@@ -5,8 +5,8 @@ import { ActionData } from 'src/types';
 export const mutations: MutationTree<TransactionStateInterface> = {
   setTransaction(state: TransactionStateInterface, transaction: ActionData) {
     state.transaction = transaction;
-    state.executed = transaction.executed;
-    if (transaction.actions.length > 0) {
+    state.executed = transaction ? transaction.executed || false : false;
+    if (transaction && transaction.actions && transaction.actions.length > 0) {
```

Comment:
I believe you can clean up this logic (lines 7-9) by checking if transaction exists at the beginning?

Reply:
Made the change.